packages/wrangler/src/constellation/createProject.tsx

```diff
@@ -0,0 +1,64 @@
+import { Text, Box } from "ink";
+import React from "react";
+import { fetchResult } from "../cfetch";
+import { withConfig } from "../config";
+import { logger } from "../logger";
+import { requireAuth } from "../user";
+import { renderToString } from "../utils/render";
+import { constellationBetaWarning } from "./utils";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+import type { Project } from "./types";
+
+export function Options(yargs: CommonYargsArgv) {
```

Comment:
To fit in with the conventions of the rest of the codebase, could non-type variables/exports be changed to camelCase, keeping PascalCase only for types and classes?

Reply:
Done.